pytorch_pfn_extras/utils/comparer.py

```diff
@@ -93,6 +93,7 @@ class OutputsComparer:
     def __init__(
             self, engines, to_compare_keys=None, *,
             compare_fn=_default_comparer,
+            max_pool=None,
```

Comment:
Maybe naming it `concurrency` is better?
```suggestion
            concurrency=None,
```